src/calendar/js/calendar.js

```diff
@@ -102,6 +102,22 @@ Y.Calendar = Y.extend(Calendar, Y.CalendarBase, {
         event.preventDefault();
     },
 
+    /**
+     * Prevent button focus on tab navigation, enable on arrow navigation
+     * @method _setTabIndex
+     * @param node {Node} The parent node
+     * @param tabIndexValue {Number} Corresponding to the value to be updated in the tabindex
+     * @protected
+     */
+    _setTabIndex : function (node, tabIndexValue) {
+        var buttonChild = node.querySelector('button');
```

Comment:
I don't think querySelector will work here. The node isn't a native node, it's a YUI node, which has different methods available. I think you'll want to do node.one('button'); or something like that.

Reply:
I agree, make sense